LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/anchor.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE HTML><!-- webkit-test-runner [ ShadowRootReferenceTargetEnabled=true ] -->
```

Comment:
Generally WebKit tends to sync WPTs in a separate PR, and track the import with a separate bug. 

I don't know if you imported these using the script, but if not, running something like:

```
./Tools/Scripts/import-w3c-tests -t web-platform-tests/shadow-dom/reference-target/
```

and check the end of the import logs which will give you a message to copy/paste into the commit message.

Reply:
Ah thanks - I did indeed run the import script, but I didn't realise it was meant to be a separate PR.

Do they usually land after the code that makes them pass?

Reply:
Generally I import them with expectation files showing the failures. Get that merged and then in the implementation PR reset those expectations to show them now passing.

Reply:
Thank you!

Reply:
I've made a separate PR for the WPT tests: https://github.com/WebKit/WebKit/pull/39801

Once that lands, I'll rebase this PR.